generators/app/templates/services/passport/index.js

```diff
@@ -9,6 +9,9 @@ import { jwtSecret, masterKey } from '../../config'
 <%_ if (authMethods.indexOf('facebook') !== -1) { _%>
 import { getMe } from '../facebook'
 <%_ } _%>
+<%_ if (authMethods.indexOf('github') !== -1) { _%>
+import { getMe } from '../github'
+<%_ } _%>
```

Comment:
It will create a conflict with `getMe` from `facebook`. You can just `import github from '../github'` and use `github.getMe` instead. We need to do this to the facebook import too.